comm_mode_browser_visual.html

```diff
@@ -0,0 +1,126 @@
+    <title>CommDebugMode Module Tree</title>
+    <style>
+        ul, #tree-container {
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+        }
+
+        .caret {
+            cursor: pointer;
+            user-select: none;
+        }
+
+        .caret::before {
+            content: "\25B6";
+            color:black;
+            display: inline-block;
+            margin-right: 6px;
+        }
+
+        .caret-down::before {
+            transform: rotate(90deg);
+        }
+
+        .tree {
+            padding-left: 20px;
+        }
+
+        .tree ul {
+            padding-left: 20px;
+        }
+
+        /* hide nested list when component is not active*/
+        .nested {
+            display: none;
+        }
+
+        /* show nested list when pressed caret pointer makes component active*/
+        .active {
+            display: block;
+        }
+
+        .forward-pass,
+        .backward-pass {
+            margin-left: 40px;
+        }
+
+        .forward-pass table {
+            margin-left: 40px;
+            width: auto;
+        }
+
+        .forward-pass table td, .forward-pass table th {
+            padding: 8px;
+        }
+
+        .forward-pass ul {
+            display: none;
+        }
+
+        table {
+            font-family: arial, sans-serif;
+            border-collapse: collapse;
+            width: 100%;
+        }
+
+        td, th {
+            border: 1px solid #dddddd;
+            text-align: left;
+            padding: 8px;
+        }
+
+        tr:nth-child(even) {
+            background-color: #dddddd;
+        }
+        #drop-area {
+            position: relative;
+            width: 25%;
+            height: 100px;
+            border: 2px dashed #ccc;
+            border-radius: 5px;
+            padding: 0px;
+            text-align: center;
+        }
+
+        .drag-drop-block {
+            display: inline-block;
+            width: 200px; /* reduce the width */
+            height: 50px; /* reduce the height */
+            background-color: #f7f7f7;
+            border: 1px solid #ccc;
+            border-radius: 5px;
+            padding: 10px; /* reduce the padding */
+            font-size: 14px; /* reduce the font size */
+            color: #666;
+            cursor: pointer;
+        }
+
+        #file-input {
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            opacity: 0;
+        }
+  </style>
+</head>
+<body>
+
+    <div id="drop-area">
+        <div class="drag-drop-block">
+          <span>Drag file here</span>
+        </div>
+        <input type="file" id="file-input" accept=".json">
+      </div>
+    <div id="tree-container"></div>
+
+    <script src="https://cdn.jsdelivr.net/gh/pytorch/pytorch@main/torch/distributed/_tensor/debug/comm_mode_broswer_visual.js"></script>
```

Comment:
What's wrong with referencing github script directly?
```suggestion
    <script src="https://raw.githubusercontent.com/pytorch/pytorch/main/torch/distributed/_tensor/debug/comm_mode_broswer_visual.js"></script>
```